metadata-ingestion/tests/unit/fivetran/test_fivetran_query.py

```diff
@@ -0,0 +1,79 @@
+from datahub.ingestion.source.fivetran.config import (
+    MAX_COLUMN_LINEAGE_PER_CONNECTOR_DEFAULT,
+    MAX_JOBS_PER_CONNECTOR_DEFAULT,
+    MAX_TABLE_LINEAGE_PER_CONNECTOR_DEFAULT,
+)
+from datahub.ingestion.source.fivetran.fivetran_query import FivetranLogQuery
+
+
+class TestFivetranLogQuery:
+    """Test cases for FivetranLogQuery with configurable limits."""
+
+    def test_fivetran_log_query_default_limits(self):
+        """Test that FivetranLogQuery uses default limits when initialized with defaults."""
+        query = FivetranLogQuery()
+
+        assert query.max_jobs_per_connector == MAX_JOBS_PER_CONNECTOR_DEFAULT
+        assert (
+            query.max_table_lineage_per_connector
+            == MAX_TABLE_LINEAGE_PER_CONNECTOR_DEFAULT
+        )
+        assert (
+            query.max_column_lineage_per_connector
+            == MAX_COLUMN_LINEAGE_PER_CONNECTOR_DEFAULT
+        )
+
+    def test_fivetran_log_query_custom_limits(self):
+        """Test that FivetranLogQuery uses custom limits when provided."""
+        query = FivetranLogQuery(
+            max_jobs_per_connector=1234,
+            max_table_lineage_per_connector=567,
+            max_column_lineage_per_connector=8901,
+        )
+
+        assert query.max_jobs_per_connector == 1234
+        assert query.max_table_lineage_per_connector == 567
+        assert query.max_column_lineage_per_connector == 8901
```

Comment:
In my opinion those 2 tests are not necessary

Reply:
Removing...